wikicharts/wikicharts.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #400.        def annotate_last_point(self, x, y, label_color='black', ha='right', xytext = (-10,0)):
</pre><p>I see why you did this, but I feel like this is very confusing, because there is already a function being used in other charts for annotating the last point (<code>annotate</code>; it would be better if <em>that</em> were named <code>annotate_last_point</code>). </p><p> </p><p>What I would suggest instead is that you continue to use <code>annotate</code> to print the value that goes right next to the last point (in this case, the value of that point) and add an <code>annotate_mean/average</code> function that adds the mean/average. This would also help address issues like the fact that using <code>annotate</code> for the average in the gender chart leads to the average being printed at the level of the last point, and not at the average. </p><p> </p><p>I know there might be some issues with the two overlapping, but I would suggest handling that in <code>annotate_mean</code> (being it's going to be used in fewer places, so it's the easier place to add something new). We can check to see what the last value is and, if the mean is close, move the mean annotation. </p><p> </p><p>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>